### PR TITLE
Add Boring Cyborg App

### PR DESCRIPTION
**Pull request by kaxil**
##### Checklist

Repo: https://github.com/kaxil/boring-cyborg
- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed.
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: {explain the action here}
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Is open source]
- [x] [Has a license]
- [x] [Has a code of conduct]
- [x] Has someone willing to at least minimally maintain them for the near future: {Kaxil}

-----
[View rendered _apps/boring-cyborg.md](https://github.com/kaxil/probot.github.io/blob/patch-2/_apps/boring-cyborg.md)

### CODE REVIEW

File: _apps/boring-cyborg.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Boring Cyborg
+description: Automatically label PRs based on the path of the file that are modified in the PR
+slug: boring-cyborg
+screenshots:
+- https://github.com/kaxil/boring-cyborg/blob/master/assets/usage-screenshot-1.png
+authors: [ "kaxil" ]
+repository: kaxil/boring-cyborg
+host: https://gentle-mesa-48041.herokuapp.com/
```
Comment (gr2m) [inline _apps/boring-cyborg.md]: This looks like a temporary URL. We have a check in place that pings this URL regularly to make sure the app is still working. Do you plan to change it anytime soon?

Comment (kaxil) [reply _apps/boring-cyborg.md]: No, the app is deployed on Heroku. Currently it is used by Apache Airflow project. 

I will definitely update the url in this markdown when I change the url.

**Comment by kaxil** [comment]
@gr2m Anything I can help to get this merged. I am also not sure why the test is failing with status code of 502. https://gentle-mesa-48041.herokuapp.com - It is a working URL

File: _apps/boring-cyborg.md (hunk 1 above)
Comment (gr2m) [reply _apps/boring-cyborg.md]: Perfect, thanks!

**Pull request merged**